.github/workflows/cd_dev.yaml

```diff
@@ -13,10 +13,10 @@ on:
 #    branches: [master]
 #  pull_request:
 #    branches: [master]
-  workflow_run:
-    workflows: [Publish]
-    branches: [master]
-    types: [completed]
+
+  repository_dispatch:
+    types: [ publish-success ]
```

Comment:
push event is more reliable since it will only be triggered on commit

Reply:
I will try that next if this does not work, but I think we tried other methods because publish/cd_dev were misfiring - but maybe that was because of the merge_queue config